docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM openjdk:8u292-jre-slim-buster
+
+LABEL maintainer="support@openboxes.com"
+EXPOSE 8080
+ENV JAVA_TOOL_OPTIONS="-Xms1024m -Xmx1024m -XX:+UseParallelGC -Djava.awt.headless=true"
```

Comment:
@jmiranda would you say that these default JVM heap size settings look alright?

Reply:
I think we also need to add PermSize = 128m or 256m. The default memory settings look ok. I don't know if the default should be the minimum (512m) or recommended (2GB), so I think 1GB is a good happy medium.

Reply:
PermSize got removed in Java 8, which OB Grails 3 uses: https://stackoverflow.com/a/22509753/2572524

Reply:
Ah, I always thought that error just meant that you could only set PermSize and not MaxPermSize. Didn't realize they replaced permanent generation with metaspace. I'll need to read up on that.